### PR TITLE
Update README.md more info from docs

### PR DESCRIPTION
**Pull request by sguequierre**
Merging further into docs readme-- adding calibration info that was missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,50 @@ This module implements two methods of the [movement sensor API](https://docs.via
 
 Please note that `GetLinearVelocity()` returns an estimation of the instantaneous linear velocity **without scale factor**. Hence, units should not be trusted and `GetLinearVelocity()` should serve as direction estimation.
 
+While `monocular-visual-odometry` enables you to add movement sensing abilities to your robot without needing specialized hardware, a dedicated [movement sensor](https://docs.viam.com/components/movement-sensor/) will generally provide more accurate readings.
+If your robot requires precise awareness of its location and its movement, you should consider using a dedicated movement sensor in addition to the `monocular-visual-odometry` module.
+
```
Comment (Rob1in) [inline README.md:18]: This not completely accurate, [see here](https://github.com/viamrobotics/docs/pull/1691#discussion_r1320047180) and [here](https://github.com/viamrobotics/docs/pull/1691#pullrequestreview-1617788404) the changes

File: README.md [hunk 2]
```diff
@@ -13,14 +13,50 @@ This module implements two methods of the [movement sensor API](https://docs.via
...
+## Configure and calibrate your camera
 
 > [!NOTE]  
-> Before configuring your movement sensor, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+> Before configuring your camera, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+
+Follow [these instructions](https://docs.viam.com/components/camera/) to configure a camera on your machine.
+Once you have configured a `camera` component, you need to calibrate it.
+Because the `monocular-visual-odometry` module performs visual odometry calculations, its visual data source (the camera) must be as well defined as possible.
+These calibration steps ensure that the video stream data that reaches the module is as uniform as possible when calculating measurements.
```
Comment (npentrel) [inline README.md:31]:
```suggestion
The following calibration steps ensure that the video stream data that reaches the module is as uniform as possible when calculating measurements.
```

File: README.md [hunk 3]
```diff
@@ -13,14 +13,50 @@ This module implements two methods of the [movement sensor API](https://docs.via
```
Comment (npentrel) [inline README.md:13]: Only the first sentence in the requirements section is a requirement. The rest should go into the introduction.

Comment (npentrel) [reply README.md:13]: And line 2 can be improved with the intro content from the docs page:

```
A `monocular-visual-odometry` [modular resource](https://docs.viam.com/extend/modular-resources/) which uses monocular visual odometry to enable any calibrated camera to function as a movement sensor. In this way, you can add basic movement sensing to your camera-equipped robot without needing a dedicated hardware movement sensor.
```

**Review by npentrel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sguequierre) [reply README.md:18]: Removing

File: README.md (hunk 3 above)
Comment (sguequierre) [reply README.md:9]: Done

File: README.md (hunk 2 above)
Comment (sguequierre) [reply README.md:28]: Done

**Review by Rob1in — APPROVED** [review]

**Pull request merged**